src/promo-manager/core/condition/condition-helpers.ts

```diff
@@ -21,6 +23,22 @@ export const ShowOnceForPeriod = (interval: DurationParam) => {
     };
 };
 
+export const ShowFinishedOnceForPeriod = (interval: DurationParam) => {
```

Comment:
Default behavior now - allow rerun finished promo. Looks like this helper works same as `ShowOnceForPeriod`